soyuz_app/views/web.py

```diff
@@ -594,6 +607,72 @@ def sectionless_assign(request):
     )
 
 
+@staff_member_required
+@require_POST
+def choose_section_leader(request):
+    section_leader_id = request.POST.get("section_leader_id")
+    section_id = request.POST.get("section_id")
+    batch_id = request.POST.get("batch_id")
+    batch = Batch.objects.get(id=batch_id)
+    batch_number = batch.number
+    course_name = batch.course.name
+    print('section leader id', section_leader_id)
+    print('section id', section_id)
+
+    # get section
+    section = Section.objects.get(id=int(section_id))
+    # get section leader
+    new_section_leader = get_user_model().objects.get(id=int(section_leader_id))
+    # assign section leader to section / update section leader
+    section.section_leader = new_section_leader
+    section.save()
+
+    return redirect("soyuz_app:get_sections", course_name=course_name, batch_number=batch_number)
+
+
+@staff_member_required
+@require_POST
+def create_zoom_room(request):
+    batch_id = request.POST.get("batch_id")
+    batch = Batch.objects.get(id=batch_id)
+    batch_number = batch.number
+    course_name = batch.course.name
+
+    sections = Section.objects.all()
+    for section in sections:
+        host = section.section_leader
+        # from zoom.py, creates zoom room and returns meeting id
+        # TODO: remove this if condition
+        if host.email != 'sam@rocketacademy.co':
```

Comment:
what does this do?

Reply:
that needs to be removed, i used it for testing because sam is not a user on rocket's account yet